site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -197,6 +197,9 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.maxDeletionParallelism`               | Maximum number of parallelizing deletion calls in GSS                                                                                                                                                                               | `64`    |
 | `agones.controller.maxGameServerDeletionsPerBatch`       | Maximum number of GameServer deletion calls per batch                                                                                                                                                                               | `64`    |
 | `agones.controller.maxPodPendingCount`                   | Maximum number of pending pods per game server set                                                                                                                                                                                  | `5000`  |
+{{% feature publishVersion="1.51.0" %}}
```

Comment:
This is where docs are great - because I didn't notice the nesting of the helm value.

So two thoughts:

1. I think this param should be `agones.sdkserver.requestsRateLimit`
2. I think let's create a new doc section of "SDK Server" to put this in.

Since the value is on the SDK Server (I know it goes through the controller to do it, but I'm thinking of what it _does_ rather than _how it does it_) -- does that make sense?